release-notes.rst

```diff
@@ -1,14 +1,25 @@
 Release Notes
 =============
 
+Version 0.2.0b1
```

Comment:
Usually release notes of beta release are set under the version 0.2.0, no need to add b1 in the release notes. Then all changes are cummulative until the final 0.2